projects/find/find.py

```diff
@@ -0,0 +1,32 @@
+import os
+import sys
+import argparse
+
+"""This method is created to find all files and directories, given a specific path. Optionally, your can use -name flag
+to search only the files that match to a specific name"""
+
+
+def find(path, name=None, out=sys.stdout):
+    if not os.path.exists(path):
+        raise ValueError("Path doesn't exist or wrong: please try again")
+    stack = [path]
+    while len(stack) > 0:
+        stack_element = stack.pop(0)
```

Comment:
there is a flaw here: below you correctly append at the end, but here you are popping from the head, which returns the oldest element in the stack, not the newest (as per LIFO rules)